basketball_reference_web_scraper/html.py

```diff
@@ -870,6 +870,17 @@ def game_url_paths(self):
         game_links = self.html.xpath(self.game_url_paths_query)
         return [game_link.attrib['href'] for game_link in game_links]
 
+class TeamRoster:
```

Comment:
Can we create a `SeasonTeamPage` class that has a property called `@team_roster_table`?

Most of all the client methods refer to a top-level `Page` class that might expose underlying tables with properties or methods.